### PR TITLE
Tomlkit 0.11.1 2022 07 26

### PR DESCRIPTION
**Pull request by ryanskeith**
Simple update to latest version.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +28,6 @@ requirements:
     - pip
     - poetry-core >=1.0.0a9
     - python
-    - setuptools
```
Comment (varlackc) [inline recipe/meta.yaml:30]: I am curious to know why `setuptools` was removed

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tomlkit-{{ version }}.tar.gz
-  sha256: ebd982d61446af95a1e082b103e250cb9e6d152eae2581d4a07d31a70b34ab0f
+  sha256: 61901f81ff4017951119cd0d1ed9b7af31c821d6845c8c477587bbdcd5e5854e
 
 build:
   number: 0
+  skip: True  # [py<36]
   noarch: python
```
Comment (varlackc) [inline recipe/meta.yaml]: We are currently not building `noarch` packages. The package will have to be built as an `architecture specific` packages.

**Review by varlackc — COMMENTED** [review]
There are some details that will have to be reviewed closely

File: recipe/meta.yaml (hunk 1 above)
Comment (ryanskeith) [reply recipe/meta.yaml:30]: @skupr-anaconda mentioned this in the 0.10.2 feedstock here:
https://github.com/AnacondaRecipes/tomlkit-feedstock/pull/8

And cited the pyproject.toml here:
https://github.com/sdispater/tomlkit/blob/e15bb79401517ca28013454662741ca9b0646fab/pyproject.toml#L61

Comment (skupr-anaconda) [reply recipe/meta.yaml:30]: @varlackc Yeah, there is another build backend - `poetry`, not `setuptools`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -27,7 +27,6 @@ requirements:
     - pip
     - poetry-core >=1.0.0a9
     - python
-    - setuptools
     - wheel
   run:
     - python >=3.6
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: If we skip `py<36` https://github.com/AnacondaRecipes/tomlkit-feedstock/pull/9/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR14 then we can use a plain `python` without pinning in `run`.

**Pull request merged**